classtime.js

```diff
@@ -214,17 +325,25 @@ function updateVariables() {
 }
 
 function updateText() {
-    document.getElementById('timeToEndOfClass').innerHTML =  getTimeToEndOfCurrentClassString()
-    document.getElementById("nextClass").innerHTML = getClassName(currentClassPeriodIndex+1)
-    document.getElementById("currentClass").innerHTML = getClassName(currentClassPeriodIndex)
-    //document.getElementById('sentence').innerHTML = getSummaryString()
     document.getElementById('time').innerHTML = getCurrentTimeString();
     document.getElementById('date').innerHTML = getCurrentDateString();
+    
+    if (currentClassPeriodIndex == -1) {
+        document.getElementById("schedule").innerHTML = "and there's <strong>no class</strong> today!"
+        $('.schoolDays').css('display', 'none')
+        return
+    }
+
+    $('.schoolDays').css('display', 'inline')
     document.getElementById("schedule").innerHTML = "You are viewing the <strong>" + getCurrentScheduleName() + "</strong> schedule."
+    document.getElementById('timeToEndOfClass').innerHTML =  getTimeToEndOfCurrentClassString()
+    document.getElementById("nextClass").innerHTML = getClassName(currentClassPeriodIndex+1)
+    document.getElementById("currentClass").innerHTML = getClassName(currentClassPeriodIndex)
 }
 
 function getCurrentScheduleName() {
```

Comment:
this method is only ever called if the schedule index is not -1, so adding a `return "no class"` wont do anything